src/uu/test/src/test.rs

```diff
@@ -10,12 +10,81 @@
 
 mod parser;
 
-use clap::{App, AppSettings};
+use clap::{crate_version, App, AppSettings};
 use parser::{parse, Symbol};
 use std::ffi::{OsStr, OsString};
 use std::path::Path;
 use uucore::executable;
 
+const USAGE: &str = "test EXPRESSION
+or:  test
+or:  [ EXPRESSION ]
+or:  [ ]
+or:  [ OPTION";
+
+const AFTER_HELP: &str = "
+Exit with the status determined by EXPRESSION.
+
+An omitted EXPRESSION defaults to false.  Otherwise,
+EXPRESSION is true or false and sets exit status.  It is one of:
+
+  ( EXPRESSION )               EXPRESSION is true
```

Comment:
all this will replace the current help mgmt or add to it?

Reply:
test does not handle the help flag at all. Only when installed as [ it should handle --help and --version, which I'm adding here. This is basically the first help mgmt test got, there was none before.

Reply:
sounds great, thanks :)